content/dev/dev-guide.md

```diff
@@ -0,0 +1,264 @@
+  - ```bash
+    cd ~/urbit-dev/zod
+    rm -r hello/*
+    ```
+- Next we'll copy the contents of the `hello` desk we created earlier. We'll need to use `cp -LR` to resolve all the symlinks.
+  - ```bash
+    cp -LR ~/urbit-git/pkg/hello/* hello/
+    ```
+- Back in the `zod` dojo we can commit the changes and install the desk.
+  - ```bash
+    |commit %hello
+    |install our %hello
+    ```
+  - You'll see `docket: awaiting manual glob for %hello desk` this is because our `desk.docket-0` files includes a `%glob-ames` clause which specifies our ship as the source, so it's waiting for us to upload the glob.
+- Open a browser and navigate to `http://localhost:80` and login with the default fake zod code `lidlut-tabwed-pillex-ridrup`. You'll see our tile, but it says installing with a spinner due to the missing glob.
+##### Create Files for Glob
+We'll now create the files for the glob. We'll use a very simple static HTML page that just displayes "Hello World!" and an image. Typically we'd have a more complex JS web app that talked to apps on our ship through Eyre's channel system, but for the sake of simplicity we'll forgo that. Let's hop back in the Unix terminal.
+ - ```bash
+   cd ~
+   mkdir hello-glob
+   cd hello-glob
+   mkdir img
+   wget -P img https://media.urbit.org/docs/userspace/dist/pot.svg
+   ```
+- We've grabbed an image to use in our "Hello world!" page. The next thing we need to add is an `index.html` file in the root of the folder. The `index.html` file is mandatory; it's what will be loaded when the app's tile is clicked. Let's open our preferred editor and create it.
+  - ```bash
+    nano index.html
+    ```
+- Paste the following into the editor and save it.
+  - ```html
+    <!DOCTYPE html> <html> <head> <style> div { text-align: center; }
+    </style> </head> <title>Hello World</title> <body> <div>
+    <h1>Hello World!</h1> <img src="img/pot.svg" alt="pot" width="219"
+    height="196" /> </div> </body> </html>
+    ```
+##### Upload to Glob
+We can now create a glob from the directory. To do so, navigate to http://localhost:80/docket/upload in the browser. This will bring up the %docket app's Globulator tool.
+
+- Simply select the `hello` desk from the drop-down, click `Choose file` and select the `hello-glob` folder in the the file browser, then hit `glob!`.
+- Now if we return to our ship's homescreen, we should see the tile looks as we specified in the docket file: http://localhost:80
+- And if we click on the tile, it'll load the index.html in our glob.
+- Our app is working!
+##### Publish
+The final step is publishing our desk with the `%treaty` agent so others can install it. To do this, there's a simple command in the dojo.
+ - ```bash
+   :treaty|publish %hello
+   ```
+   - Note: For desks without a docket file (and therefore without a tile and glob), treaty can't be used. Instead you can make the desk public with `|public %desk-name`.
```

Comment:
Should clarify further and explain that desks aren't publicly readable by default, and that `:treaty|publish` not only publishes the metadata, but also makes the desk publicly readable, like `|public`.